docker-build/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu
+
+# LABEL about image
+LABEL maintainer="mhuang7447@sdsu.edu"
+LABEL description="BDA696"
+
+RUN apt-get update && apt install mariadb-client -y
+
+WORKDIR /home/mu-ting/docker-build
+
+COPY ./SQLBASH.sh /home/mu-ting/docker-build/SQLBASH.sh
+COPY baseball.sql /home/mu-ting/docker-build/baseball.sql
+
+RUN chmod +x SQLBASH.sh
+
+
+VOLUME ["/home/mu-ting/docker-build","SQLBASH.sh","baseball.sql"]
+
+CMD ["/home/mu-ting/docker-build","baseball.sql","SQLBASH.sh"]
```

Comment:
When I try to run docker-compose up I get this error:
ERROR: for sql_service  Cannot start service sql_service: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"/home/mu-ting/docker-build\": permission denied": unknown

I don't think this has anything to do with a directory issue. It should be creating that directory in the container and it makes it past the WORKDIR step on line 9. 

If you have any hints that I might be doing wrong please tell me so I can actually get this to run. I think it's more of an issue on my end, because just by looking at it I can't figure out what wouldn't be working.

Reply:
I think you should update the PATH where you save this file. I saved the files in this path so try update the PATH